people_recognition_msgs/msg/Person2D.msg

```diff
@@ -1,8 +0,0 @@
-string name
-uint8 age
-uint8 gender
```

Comment:
Please define an [enum/constants](https://wiki.ros.org/action/show/msg?action=show&redirect=ROS%2FMessage_Description_Language#Constants) for the genders? Eg. FEMALE=0, MALE=1 ?